src/zenml/new/pipelines/pipeline.py

```diff
@@ -691,7 +675,21 @@ def _run(
                         f"infrastructure.",
                     )
 
-    get_runs = GetRunsDescriptor()
+    def get_runs(self, **kwargs: Any) -> List[PipelineRunResponseModel]:
```

Comment:
This means `PipelineClass.get_runs(...)` is removed in a non-backwards compatible way right?
I don't think that's a problem (and people using the new `@pipeline` decorator don't get pipeline classes anymore anyway), just writing this here so we don't forget to include it in the release notes

Reply:
Yeah this PR actually contains a lot of non-backwards compatible changes (see "Renamed" / "Removed" sections of PR description), so this is definitely a breaking change 👍